### ISSUE TITLE
Number of people low vision vs blind

### ISSUE THREAD

**Issue by shawna-slh** [w3c/low-vision-a11y-tf#32]
http://w3c.github.io/low-vision-a11y-tf/requirements.html#incidence
current wording:

<blockquote>In a fact sheet on visual impairment and blindness [WHO-VI] the World Health Organization (WHO) estimates that there are 246 million people worldwide who have low vision and 39 million people are blind, indicating that 86% of the people with visual impairments have low vision.</blockquote>

[Section 2.2 Incidence](http://w3c.github.io/low-vision-a11y-tf/requirements.html#incidence) - I wonder if it would be relevant to also add information about the ratio of people with low vision, as opposed to completely blind. I believe there is value in comparison, especially when it comes to advocacy, because we often have to debunk the myth that accessibility is for people who are blind. Demonstrating the ratio between low vision users ans blind users always has a lasting effect on the people I teach to. I also feel like that sentence is too long. If you agree, then I would recommend the following addition/changes:

"In a fact sheet on visual impairment and blindness [WHO-VI], the World Health Organization (WHO) estimates that there are 246 million people worldwide who have low vision and 39 million people are blind. This indicates that 86% of the people with visual impairments have low vision. This means there are roughly 6.31 times as many people who have low vision than people who are blind.”

from Denis Boudreau https://lists.w3.org/Archives/Public/public-low-vision-comments/2016JanMar/0003.html

**Comment by awkawk** [comment]
I don't understand the issue.  I think that saying that 86% of people with visual impairments is more meaningful than saying there are 6.31 times the number of LV people compared to blind people.

How about:
The World Health Organization (WHO) estimates that there are 246 million people worldwide who have low vision and 39 million people are blind (reference link), indicating that 86% of the people with visual impairments have low vision.

**Comment by dboudreau** [comment]
Andrew, 

I’m good with either. I was just trying to come up with an impactful way to demonstrate the huge gap between the two groups.

/Denis

**Comment by awkawk** [comment]
@dboudreau right, I get it.  Thanks for the comment. I'd like to not get too deep into the numbers argument since that is often a losing one, but I agree that we want to emphasize that low-vision users are a large group.

**Comment by shawna-slh** [comment]
I think we can get the point across better with fewer words by adding the percent who are blind so there's a clear comparison, e.g.:

<blockquote>… (WHO) estimates that there are 246 million people worldwide who have low vision and 39 million people are blind, indicating that generally 16% of people with visual impairments are blind and 84% have low vision.</blockquote>

(The first part of the sentence was a change Michael Cooper made for the reference. I think that's an editor thing that the whole group doesn't need to spend time discussing right now. I will consider it and propose a solution.)

**Comment by shawna-slh** [comment]
Discussed [30 March telecon](https://www.w3.org/2016/03/30-lvtf-minutes.html#item06)
+1s to including only the percentages, and not the times

Action: Jim confirm numbers.
Done! [Jim's confirmation of numbers e-mail](https://lists.w3.org/Archives/Public/public-low-vision-a11y-tf/2016Mar/0095.html)

**Comment by shawna-slh** [comment]
Looking at it again with [Jim's info](https://lists.w3.org/Archives/Public/public-low-vision-a11y-tf/2016Mar/0095.html) and rechecking the original source, I think it makes sense to include the total and the breakdown. I suggest:

> ...(WHO) estimates that of the 285 million people worldwide who are visually impaired, 39 million are blind and 246 have low vision; that is, 14% are blind and 86% have low vision.

I think that more clearly makes the main point that we want to get across &mdash; that "visual impairment" includes many more people with low vision than blind (and it's easier to check the % if anyone is so inclined:).

**Comment by slhenry** [comment]
asked Denis if OK [4 April e-mail](https://lists.w3.org/Archives/Public/public-low-vision-comments/2016AprJun/0000.html)

**Comment by shawna-slh** [comment]
asked @dboudreau if OK [4 April e-mail](https://lists.w3.org/Archives/Public/public-low-vision-comments/2016AprJun/0000.html)

**Comment by WayneEDick** [comment]
I have looked for my resource on LV demographics that shows a flat curve from birth to 40, a gradual increase to 60 with a steep growth past 65.  

Wayne

**Comment by shawna-slh** [comment]
from discussion with @dboudreau:
- make point clear
- maybe simple graphic? ... probably not on this page

RESOLUTION
explain the issue more on https://www.w3.org/WAI/users/low-vision/

**Comment by shawna-slh** [comment]
Updated wording in User Needs / Requirements Doc.

Left open for WAI page.
